### DISCUSSION TITLE
Does gemm-streamk has stricter alignment requirement than normal gemm?

### DISCUSSION THREAD

**Discussion by lygztq** [NVIDIA/cutlass#722] [Q&A] [answered]
I find that problem sizes like (1, 512, 1024) (row, col -> row) will fail when using gemm-streamk, but when using normal gemm, this will not happen. So does gemm-streamk has stricter alignment requirement than normal gemm?

**Comment by jackkosaian** [comment, marked as answer]
Hi, @lygztq. Does [this PR](https://github.com/NVIDIA/cutlass/pull/717) help?

**Reply by lygztq** [reply]
Thanks! This PR solved the problem exactly I have.
